pkg/appmgmt/general/general.go

```diff
@@ -162,9 +168,14 @@ func (os *Manager) getAppMetadata(pod *v1.Pod) (interfaces.ApplicationMetadata,
 		TaskGroups:                 taskGroups,
 		OwnerReferences:            ownerReferences,
 		SchedulingPolicyParameters: schedulingPolicyParams,
+		CreationTime:               creationTime,
 	}, true
 }
 
+func (os *Manager) getAppMetadata(pod *v1.Pod) (interfaces.ApplicationMetadata, bool) {
```

Comment:
I believe it would be cleaner to have only this method with (pod, bool) arguments, but for now I wanted to avoid too many changes. This is a potential TODO.